src/combineReducers.js

```diff
@@ -74,6 +74,52 @@ function assertReducerSanity(reducers) {
   })
 }
 
+function getFinalReducers(reducers) {
+  var functionReducers = {}
+
+  Object.keys(reducers).map(key => {
+    if (typeof reducers[key] === 'function') {
+      functionReducers[key] = reducers[key]
+    }
+  })
+
+  return functionReducers
+}
+
+function checkStateChange(keys, finalReducers, state, action) {
+  var hasChanged = false
+  var nextState = {}
+  keys.map(key => {
```

Comment:
Let’s use the `for` loop here. If you look at the history, it was `map` in the beginning. We landed at the current version after a few optimization passes so it’s unwise to go back now. 😉